sdp.go

```diff
@@ -12,23 +13,52 @@ import (
 	"github.com/pion/sdp/v2"
 )
 
+const (
+	// TODO(sgotti) add these to pion/sdp
+	extMapValueSDESMid         = 4
+	extMapValueSDESRTPStreamID = 5
```

Comment:
Is this the value that Chrome just happens to use for these extensions or is fixed in spec?

Reply:
I used the values like the one chrome used, but they could just be random. The spec doesn't provide default values.

Reply:
Right, they should be determined from the offer per media description. Implementations I've seen start the extmap values at 1 and count up for each of the extensions they offer.

Reply:
We could just do it, it really doesn't matter. We could also use random numbers every time...

Currently I'm using pion/sdp  ExtMapValueTransportCC that has a value of 3 (don't know why and probably I started with 4 for that reason).

I'll just propose to remove `sdp.WithTransportCCExtmap` function that's not used anymore and IMHO wrong.